cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/CVRUtilities.java

```diff
@@ -49,6 +50,7 @@
 public class CVRUtilities {
     private String METADATA_PREFIX = "#";
     private String DELIMITER = "\t";
+    private static final Pattern WHITESPACE_PATTERN = Pattern.compile("^[\\s]+|[\\s]+$|[\\s]+");
```

Comment:
3 regex patterns that were all called one after another were merged into 1 regex and the regex was compiled once and then re-used.